src/net/sourceforge/kolmafia/session/BastilleBattalionManager.java

```diff
@@ -0,0 +1,915 @@
+package net.sourceforge.kolmafia.session;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import net.sourceforge.kolmafia.AdventureResult;
+import net.sourceforge.kolmafia.KoLConstants;
+import net.sourceforge.kolmafia.RequestLogger;
+import net.sourceforge.kolmafia.objectpool.EffectPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import net.sourceforge.kolmafia.request.GenericRequest;
+import net.sourceforge.kolmafia.utilities.ChoiceUtilities;
+import net.sourceforge.kolmafia.utilities.StringUtilities;
+
+public class BastilleBattalionManager {
```

Comment:
trivial: As in #530, if you could provide a private constructor that would prevent codecov wanting coverage instantiating this class.

Reply:
I had no idea. Done.